src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].toString().toLowerCase();

```

Comment:
```suggestion
        return Color.values()[index].name().toLowerCase();
```
when you want to extract enum name you should use name() method

Reply:
Thanks, I will know now